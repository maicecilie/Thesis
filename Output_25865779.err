Loaded dependency [python3/3.11.9]: sqlite3/3.45.3
Loaded dependency [python3/3.11.9]: gcc/12.3.0-binutils-2.40
Loaded module: python3/3.11.9

Loading python3/3.11.9
  Loading requirement: sqlite3/3.45.3 gcc/12.3.0-binutils-2.40
Loaded dependency [numpy/1.26.4-python-3.11.9-openblas-0.3.27]: openblas/0.3.27
Loaded module: numpy/1.26.4-python-3.11.9-openblas-0.3.27

Loading numpy/1.26.4-python-3.11.9-openblas-0.3.27
  Loading requirement: openblas/0.3.27
Loaded module: scipy/1.13.0-python-3.11.9
Loaded module: matplotlib/3.8.4-numpy-1.26.4-python-3.11.9
Loaded module: pandas/2.2.2-python-3.11.9
Loaded module: cuda/11.8
Loaded module: cudnn/v9.8.0.87-prod-cuda-11.X
