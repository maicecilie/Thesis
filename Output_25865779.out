Script started!
Python version: 3.11.9 (main, May 16 2024, 07:14:35) [GCC 12.3.0]
