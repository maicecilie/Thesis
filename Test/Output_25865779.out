Script started!
Python version: 3.11.9 (main, May 16 2024, 07:14:35) [GCC 12.3.0]
Epoch [1/20], Loss: 1.5720
Epoch [2/20], Loss: 0.9708
Epoch [3/20], Loss: 0.7021
Epoch [4/20], Loss: 0.5673
Epoch [5/20], Loss: 0.4772
